pulp_ansible/tests/functional/api/collection/v3/test_collection.py

```diff
@@ -196,9 +195,10 @@ def test_collection_upload(collection_upload):
     assert "finished_at" in collection_upload
     assert "messages" in collection_upload
 
-    for key, value in collection_upload.items():
-        if key in COLLECTION_METADATA.keys():
-            assert COLLECTION_METADATA[key] == value, collection_upload
+    # TODO: Add this back when namespace, name, and version are apart of the CollectionImport
+    # for key, value in collection_upload.items():
+    #     if key in COLLECTION_METADATA.keys():
+    #         assert COLLECTION_METADATA[key] == value, collection_upload
```

Comment:
If these do not assert anymore, i think we are seeing a change behavioural change unrelated to the bug / improvement advertised in the PR.

Reply:
The asserts were never being ran in the first place. `collection_upload` is a `CollectionImport` response and does not have the fields in `COLLECTION_METADATA` : `namespace, name, version`. So commenting them out does nothing I could have left it as is.

Reply:
Are you saying the keys should have been there? Then we should not guard with the "if". If not can we just remove the block completely?

Reply:
No idea if they should have been there. But they will be once I add `Namespace`s.